### PR TITLE
New package: libicns-0.8.1

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3518,3 +3518,4 @@ libmygpo-qt5.so.1 libmygpo-qt-1.1.0_1
 libluv.so.1 libluv-1.30.1.0_1
 libarmadillo.so.9 armadillo-9.700.2_1
 libvarnishapi.so.2 libvarnishapi-6.3.0_1
+libicns.so.1 libicns-0.8.1_1
```
Comment (Hoshpak) [inline common/shlibs]: there should be a line break here.

File: srcpkgs/libicns/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'libicns'
+pkgname=libicns
+version=0.8.1
+revision=1
+build_style=gnu-configure
+makedepends="libpng-devel jasper-devel"
+depends="libpng jasper"
```
Comment (Hoshpak) [inline srcpkgs/libicns/template]: this is automatically detected via shlibs and shouldn't be added explicitley.

File: common/shlibs (hunk 1 above)
Comment (teldra) [reply common/shlibs]: done

File: srcpkgs/libicns/template (hunk 2 above)
Comment (teldra) [reply srcpkgs/libicns/template]: done

**Pull request merged**